Assignment/Book-Management/src/app/core/services/category.service.ts

```diff
@@ -4,11 +4,22 @@ import { environment } from './../../../environments/environment';
 
 @Injectable({providedIn: 'root'})
 export class CategoryService {
+
+  token = "Bearer eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJuYW1laWQiOiIxIiwidW5pcXVlX25hbWUiOiJtYW5hZ2VyMSIsInJvbGUiOiJNQU5BR0VSIiwibmJmIjoxNTk1ODEyMjY2LCJleHAiOjE1OTYyNDQyNjYsImlhdCI6MTU5NTgxMjI2Nn0.7CiOI-95Vxr6PsrK3-Qo0CaZONrHefist8u8Jf4Wk-rdl68wpabj-qJN7OgdLXRbdIGlU8D3Rg8Cx4hYZuXDGw"
+
```

Comment:
Store token in the localStorage and add to the request by Interceptors.

Reply:
done